README.md

```diff
@@ -22,7 +22,9 @@ Try to find and use a system-wide installed OpenSSL library:
 cargo install cargo-geiger
 ```
 
-Or, build and statically link OpenSSL as part of the cargo-geiger executable:
+Or, use OpenSSL from the
+[openssl-sys crate](https://docs.rs/openssl/0.10.30/openssl/).
```

Comment:
I'm pretty sure `cargo` always pulls in some version of OpenSSL, the only difference is if the OpenSSL library is statically linked into the executable or if it's dynamically linked somewhere from the system. Please correct me if I'm wrong here.

Reply:
I apologize for my slow response.  I don't understand what changes to the PR your comment implies.  Would you please clarify?

Reply:
The `openssl` crate is always used by cargo, it's either dynamically linked and loaded at runtime or statically linked into the `cargo-geiger` executable as part of the `cargo` crate dependency tree.

...if I have understood how the `cargo` crate handles its dependencies.

Reply:
It's the "Or, use OpenSSL from the" that I'm talking about. It's always in use, but linked statically or dynamically. I'll go ahead and close now, but feel free to reply and keep the discussion going if you think I'm misunderstanding this PR.

Reply:
I was confused when I tried to install cargo-geiger and install command failed (https://github.com/rust-secure-code/cargo-geiger/issues/149).  I read the error message which says that it can't use the OS's OpenSSL.  This README told me how to statically link OpenSSL into the binary.  I thought it means statically link the OS's OpenSSL into the binary, which obviously won't work because it already failed to dynamically link it.  I needed the README to tell me how to use a different OpenSSL library, specifically one that is distributed as a crate.  That's why I chose the wording above.

If you think the wording is unclear, please tell me what would be better.  Write "Change this line to ...".  Or tell me clearly what you want it to say and I will try to come up with some wording that meets your needs and also would meet the needs of other macOS users who hit the installation error.

Reply:
From your comment https://github.com/rust-secure-code/cargo-geiger/issues/149#issuecomment-735421330:
> > Update the readme to state that vendored-openssl feature uses "OpenSSL from the openssl-sys crate" and builds on macOS.
>
> A PR with README.md improvements would be most welcome.

This PR makes this change.

> > Update the build error to say "Run cargo install cargo-geiger --features vendored-openssl to build with OpenSSL from the openssl-sys crate."
> 
> This would be a nice improvement for the user experience on macOS! If you would like to make a PR for this, that would be great!

Done in https://github.com/sfackler/rust-openssl/pull/1382 .

Reply:
Thanks for your patience with me on this PR.